service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -46,10 +74,84 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
             caseDetails.getData().put("hiddenOrders", hiddenOrders);
         }
 
+        if ("FPLA-2710".equals(migrationId)) {
+            CaseData caseData = getCaseData(caseDetails);
+            if (casesWithUploadDraftCMOs.contains(caseData.getId()) && isNotEmpty(caseData.getDraftUploadedCMOs())) {
```

Comment:
i would check if caseData.getHearingOrdersBundlesDrafts() is not empty. If there is new structure then i guess data was already migrated

Reply:
updated